web/server/watcher.py

```diff
@@ -28,33 +28,40 @@ async def watch_project() -> None:
         (settings.project_path / c.METRICS).resolve(),
         (settings.project_path / c.SEEDS).resolve(),
     ]
+    ignore_dirs = ensure_list(".env")
```

Comment:
This doesn't seem necessary when you can explicitly make this a list, i.e. `ignore_dirs = [".env"]`.